### ISSUE TITLE
Add repo2docker logo to Jupyter design assets repo

### ISSUE THREAD

**Issue by willingc** [jupyterhub/repo2docker#469]
Leaving this as a placeholder to check that we have provided a copy of this logo to the Jupyter design assets repo.

**Comment by manics** [comment]
Did anyone do this?

**Comment by choldgraf** [comment]
@manics I don't believe so. I believe that we still need to add the repo2docker logo here: https://github.com/jupyterhub/design

Another thing we could consider doing is creating a shared Figma board that any team member can access. That way we could more quickly create and share logos, rather than using a git repository for this.
